docs/content/en/docs-dev/user-guide/configuration-reference.md

```diff
@@ -218,6 +224,12 @@ One of `yamlField` or `regex` is required.
 | disabled | bool | Whether to exclude application from triggering target when application is at `OUT_OF_SYNC` state. Default is `true`. | No |
 | minWindow | duration | Minimum amount of time must be elapsed since the last deployment. This can be used to avoid triggering unnecessary continuous deployments based on `OUT_OF_SYNC` status. Default is `5m`. | No |
 
+## OnChain
+
+| Field | Type | Description | Required |
+|-|-|-|-|
+| disabled | bool | Whether to exclude application from triggering target when application is counted as a node of some chains. Default is `false`. | No |
```

Comment:
How about changing its default value to `true` to avoid triggered unexpected applications?

Reply:
Sure 👍 PTAL https://github.com/pipe-cd/pipe/pull/2932